FRANKDOC.md

```diff
@@ -11,8 +11,10 @@ The Frank!Doc provides reference information for Frank developers as explained i
 | JavaDoc tag | `@ff.forward` | Class | Describes a forward (e.g. `success`, `failure`). First argument is name of forward. Second argument is description.|
 | JavaDoc tag | `@ff.tag` | Class | Tag that classifies the Java class. First argument is tag name, second argument is tag value.|
 | JavaDoc tag | `@ff.default` | Attribute setter | Describes default value. |
+| Java annotation | `@DefaultValue` | Attribute setter | Does the same as `@ff.default`. |
```

Comment:
I think it would be wise to restructure the table: instead of columns 'kind' and 'name', use columns 'javadoc tag' and 'java annotation'. Then `@ff.default` and `@DefaultValue` can share their description.

Reply:
Good idea. I will update it when I have time.

Reply:
Done.